aria/cli/args_parser.py

```diff
@@ -137,10 +137,6 @@ def add_workflow_parser(workflow):
         '-w', '--workflow',
         default='install',
         help='The workflow name')
-    workflow.add_argument(
```

Comment:
It's in the context. Awkward, I know. :/ This is something we should fix outside the scope of this JIRA.